karabo_bridge/serializer.py

```diff
@@ -179,7 +179,7 @@ def deserialize(msg):
         msg = [Frame(m) for m in msg]
 
     if len(msg) < 2:  # protocol version 1.0
-        data = unpack(msg[-1].bytes)
+        data = unpack(msg[-1].bytes, object_hook=msgpack_numpy.decode)
```

Comment:
Is this something we just overlooked in the previous PR?

Reply:
It was hooking on a monkey patch I removed in the simulation code